configuration/items.md

```diff
@@ -49,7 +49,8 @@ The example above defines an item:
 * with name `Sensor_Temperature`
```

Comment:
As the item in the example has changed from Sensor_Temperature to Temperature shouldn't this be "with name `Temperature`"?